packages/create-svelte/bin.js

```diff
@@ -92,6 +92,14 @@ async function main() {
 					initial: false,
 					active: 'Yes',
 					inactive: 'No'
+				},
+				{
+					type: 'toggle',
+					name: 'playwright',
+					message: 'Add Playwright for end-to-end browser testing?',
```

Comment:
actually on second thoughts i don't think we should call this end-to-end testing @benmccann. integration testing is more accurate, but probably easier to just do this

```suggestion
					message: 'Add Playwright for browser testing?',
```

Reply:
sure. happy to avoid the debate over what the correct term is :smile: